Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -5348,6 +5343,23 @@ - (void)requestAutocorrectionContextWithCompletionHandler:(void (^)(UIWKAutocorr
         return;
     }
 
+    [self _internalRequestAutocorrectionContextWithCompletionHandler:[completionHandler = makeBlockPtr(completionHandler), strongSelf = retainPtr(self)](WebKit::RequestAutocorrectionContextResult result) mutable {
```

Comment:
Does this have to be `mutable`?

Reply:
Good catch! We don't need the `mutable`s here (this was probably a remnant from when I tried to `WTFMove` completion handlers instead of just passing the ObjC blocks through)